index.bs

```diff
@@ -1681,8 +1682,13 @@ these steps:
 
 <div class="note domintro">
   : |handle| = await window . {{Window/showDirectoryPicker()}}
+  : |handle| = await window . {{Window/showDirectoryPicker()}}({ {{DirectoryPickerOptions/mode}}: 'read' })
+  :: Shows a directory picker that lets the user select a single directory, returning a handle for
+    the selected directory if the user grants read permission.
+
+  : handles = await window . {{Window/showDirectoryPicker()}}({ {{DirectoryPickerOptions/mode}}: 'readwrite' })
   :: Shows a directory picker that lets the user select a single directory, returning a handle for
-    the selected directory.
+    the selected directory. The user agent can combine read and write permission requests on this handle into one prompt.
```

Comment:
Can we clarify that this prompt is still separate from the directory picker?

Reply:
Good point. I've added "subsequent", which I think is enough?